### PR TITLE
Us 9916 1 accessibility fixes

### PR DESCRIPTION
**Pull request by monalisasarkar123**
User Story
(US-9916-1)
1st Claim - Claimant - Sign Out of Claim Journey -> Accessibility fixes

### CODE REVIEW

**Review by IanOvenden — APPROVED** [review]
Happy for these changes to be merged.  I've been through the code and the focus trap is working as expected and with assistive tech enabled, the text is all being conveyed and in the correct order.

**Review by SushreeM-h — APPROVED** [review]

File: assets/i18n/en.json [hunk 1]
```diff
@@ -69,5 +69,6 @@
   "Day": "Day",
   "Month": "Month",
   "Year": "year",
-  "Close": "Close"
+  "Close": "Close",
+  "SIGN_OUT_MESSAGE":"this sign out message"
```
Comment (AkhilaHMRC) [inline assets/i18n/en.json:73]: is this added to cy.json ? even if we do not have the translation yet, the entry should be available in both the json files. keep the text in english if translation is not available yet

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]
Add an entry for cy.json

File: assets/i18n/en.json (hunk 1 above)
Comment (monalisasarkar123) [reply assets/i18n/en.json:73]: sure

**Review by AkhilaHMRC — APPROVED** [review]

**Pull request merged**